docs/track.rst

```diff
@@ -2464,6 +2581,90 @@ Meta-data
 
 This operation returns no meta-data.
 
+open-point-in-time
+~~~~~~~~~~~~~~~~~~
+
+.. note::
+    This operation can only be used inside a ``composite`` operation because it needs to write the Point-In-Time ID for use for later operations.
+
+With the ``open-point-in-time`` operation you can open a `point in time <https://www.elastic.co/guide/en/elasticsearch/reference/current/point-in-time-api.html>`_ to be used in subsequent `search` tasks.
+
+Properties
+""""""""""
+
+* ``name``: Referenced by other tasks to use the ``pit_id`` produced by this operation.
+* ``index`` (optional): An `index pattern <https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-index.html>`_ that defines which indices or data streams for which to create this point-in-time. Only needed if the ``indices`` or ``data-streams`` section contains more than one index or data stream respectively. Otherwise, Rally will automatically derive the index or data stream to use.
+* ``keep-alive`` (optional): Duration to keep the point-in-time open after initialization (default: ``1m``)
+
+Meta-data
+"""""""""
+
+The operation returns no meta-data.
+
+close-point-in-time
+~~~~~~~~~~~~~~~~~~~
+
+.. note::
+    This operation can only be used inside a ``composite`` operation because it needs to read the Point-In-Time ID from earlier operations.
+
+With the ``close-point-in-time`` operation you can close a previously opened point in time from another task.
+
+Properties
+""""""""""
+
+* ``with-point-in-time-from``: Denotes the ``name`` of a given ``open-point-in-time`` operation (from the same composite stream) whose `pit_id` should be closed.
+
+**Example**
+
+In this example, a point-in-time is opened, used by a ``search_after``-based search operation, and closed::
```

Comment:
i was going for a "this is what we are doing to elasticsearch" description rather than a "this is what we are doing in rally" description, but I take this hesitation as sufficient evidence it should be changed (so it was)